draft-irtf-cfrg-vdaf.md

```diff
@@ -3321,12 +3321,32 @@ struct {
 } Prio3AggShare;
 ~~~
 
-## The FLP Construction {#flp-bbcggi19}
+## FLP Construction {#flp-bbcggi19}
 
-This section describes an FLP based on the construction from {{BBCGGI19}},
-Section 4.2. We begin in {{flp-bbcggi19-overview}} with an overview of their
-proof system and some extensions to it. The construction is specified in
-{{flp-bbcggi19-construction}}.
+This section specifies an implementation of the `Flp` interface ({{flp}}) based
+on the construction from {{BBCGGI19}}, Section 4.2. We begin in
+{{flp-bbcggi19-overview}} with an overview of the proof system and some
+extensions to it. {{flp-bbcggi19-valid}} defines validity circuits, the core
+component of the proof system that defines measurement validity and how
```

Comment:
A suggestion, to avoid using "defines" twice in a clause:
```suggestion
component of the proof system that determines measurement validity and how
```